packages/fe-container/src/components/top-bar-menu/TopBarMenu.less

```diff
@@ -0,0 +1,56 @@
+@import '../../styles/_variables';
+
+.topBarMenu_container  {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+  justify-content: space-around;
+  max-height: @top-bar-height;
+
+  & .topBarMenu_entry {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    overflow: hidden;
+    cursor: pointer;
+    padding-left: 11px;
+    padding-right: 11px;
+
+    &.active {
+      background: var(--microlc-primary-color-tint-89);
+      margin-bottom: @menu-entry-active-border;
+      box-shadow: 0 @menu-entry-active-border var(--microlc-primary-color);
+
+      .topBarMenuEntry_text {
+        color: var(--microlc-primary-color);
+      }
+    }
+
+    & .topBarMenuEntry_icon {
+      height: @top-bar-height;
+      font-size: @top-bar-icon-size;
+      color: var(--microlc-primary-color);
+      display: none;
+      justify-content: center;
+      flex-direction: column;
+
+      @media (max-width: 1280px) {
```

Comment:
I recommend saving all the breackpoints in variables so as not to risk inserting custom ones between one file and another

Reply:
these are specific only for the topbar menu